docs/CODEOWNERS

```
@@ -6,4 +6,5 @@
 
 # Code owners for individual blueprints
 /blueprints/influxdb/ @colemamd
-/blueprints/unifi/ @colemamd
\ No newline at end of file
+/blueprints/unifi/ @colemamd
+/blueprints/grafana/ @colemamd
```

needs a white line at the end:
```suggestion
/blueprints/grafana/ @colemamd

```